src/App.tsx

```diff
@@ -46,7 +46,7 @@ const App = () => {
 
   useEffect(() => {
     document.documentElement.lang = locale;
-    moment.locale(locale);
+    dayjs.locale(locale);
```

Comment:
Har du sjekket om dette funker med norske språk? Tror du må importere locale-spesifikke pakker for dayjs, se https://day.js.org/docs/en/i18n/changing-locale

Reply:
Trenger vi egentlig å sette locale her i det hele tatt? 🤔 Antar at dette er for å oversette navn på måneder etc, men vi bruker vel kun tall i datoene her.

Reply:
Usikker, mulig det påvirker formattering på tid/dato også, AM/PM osv. Men vi setter kanskje formatteringen eksplisitt overalt? Hvis alt ser greit ut uten locale, så kan den sikkert droppes. 😃

Reply:
Vi setter formattering eksplisitt overalt. Så vidt jeg kunne se i prod så er dato og tidspunkt likt uavhengig av locale. Fjernet bare linja 😄